.circleci/config.yml

```diff
@@ -717,37 +707,31 @@ workflows:
           requires:
             - otp_23
           filters: *all_tags
-      # ============= DYNAMIC DOMAINS =============
       - big_tests:
-          name: dynamic_domains
+          name: pgsql_mnesia_23
           otp_package: 23.0.3-1
-          spec: dynamic_domains.spec
           preset: pgsql_mnesia
           db: pgsql
           context: mongooseim-org
           requires:
             - otp_23
           filters: *all_tags
-      # ============= 1 VERSION OLDER TESTS =============
-      - big_tests:
-          name: ldap_mnesia_22
-          otp_package: 22.3.4.9-1
-          preset: ldap_mnesia
-          db: mnesia
-          context: mongooseim-org
-          requires:
-            - otp_22
-          filters: *all_tags
       - big_tests:
-          name: pgsql_mnesia_22
-          otp_package: 22.3.4.9-1
+          name: dynamic_domains_23
+          otp_package: 23.0.3-1
+          spec: dynamic_domains.spec
           preset: pgsql_mnesia
           db: pgsql
           context: mongooseim-org
           requires:
-            - otp_22
+            - otp_23
+          filters: *all_tags
+      # ============= TESTS OTP 24 =============
+      - build:
+          name: otp_24
+          otp_package: 24.0.2-1
```

Comment:
Looks like can be use `24.0.3`. Same at line `791`.

Reply:
Wait, no, that version hasn't been built as an ubuntu package yet 😕

Reply:
Oh I see...

Reply:
Only for notes: OTP 24 have some issues with processes working, this was fixed in [24.0.3](https://github.com/erlang/otp/releases/tag/OTP-24.0.3), this can be helpful when will be needed create a image for release.

Reply:
I think we might as well update only on medium versions, like when 24.1 is released. Minor versions come too often and we would be updating our scripts as often.